theme/lynx-ui-home/index.scss

```
@@ -121,11 +129,113 @@
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 14px;
+    background: rgb(from var(--rp-c-bg) r g b / 55%);
+  }
+
+  .ui-home-compat-label {
+    font-size: 1.125rem;
+    font-weight: 400;
+    letter-spacing: -0.02em;
+    color: rgb(from currentColor r g b / 92%);
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)